### PR TITLE
Add video info

### PR DESCRIPTION
**Pull request by knylander-grafana**
Added info about the vimeo short code for embedding videos.

### CODE REVIEW

File: docs/sources/writing-guide/image-guidelines/index.md [hunk 1]
```diff
@@ -255,13 +255,28 @@ It is important that you generate a local build of your docs so that you can ver
...
+   - Path to image stored in the website repo: `static/static/img/docs/grafana-cloud/k8s-node-capacity.png`
    - Corresponding shortcode: `{{</* figure src="/static/img/docs/grafana-cloud/k8s-node-capacity.png" */>}}`.
 
 1. Run `make docs` on your branch and verify that the image appears.
 1. (Optional) Make adjustments to the image in the website repo and test again.
+
+
+## Videos
+
+The Create Services team sometimes creates videos for blog posts and other collateral. Most of these videos are hosted on Vimeo.
```
Comment (knylander-grafana) [inline docs/sources/writing-guide/image-guidelines/index.md:272]:
```suggestion
The Creative Services team periodically creates videos for blog posts and other collateral. Most of these videos are hosted on Vimeo.
```

File: docs/sources/writing-guide/image-guidelines/index.md [hunk 2]
```diff
@@ -255,13 +255,28 @@ It is important that you generate a local build of your docs so that you can ver
...
 
 1. Run `make docs` on your branch and verify that the image appears.
 1. (Optional) Make adjustments to the image in the website repo and test again.
+
+
+## Videos
+
+The Creative Services team periodically creates videos for blog posts and other collateral. Most of these videos are hosted on Vimeo.
+
+You can embed a Vimeo-hosted video by using the `vimeo` short code and the video number:
```
Comment (karengermond) [inline docs/sources/writing-guide/image-guidelines/index.md:274]: Could you show the (unevaluated) shortcode for this example?  I guess you could just put
` {{< vimeo 773194063 >}}` 
on a line by itself like in the figure example in the previous section?

**Review by karengermond — APPROVED** [review]
LGTM! Thanks for doing this.  I was wondering about that...

File: docs/sources/writing-guide/image-guidelines/index.md (hunk 2 above)
Comment (knylander-grafana) [reply docs/sources/writing-guide/image-guidelines/index.md:274]: Apparently, you can put the video link on it's own line or at the end of a sentence. It floats to where it fits.

Comment (knylander-grafana) [inline docs/sources/writing-guide/image-guidelines/index.md:274]:
```suggestion
You can embed a Vimeo-hosted video by using the `vimeo` short code and the video number: `{{< vimeo 1111111>}}`.
```

File: docs/sources/writing-guide/image-guidelines/index.md [hunk 3]
```diff
@@ -255,13 +255,28 @@ It is important that you generate a local build of your docs so that you can ver
...
+
+```
+{{< vimeo 773194063 >}}
+```
+
+In this example, the video is a Preview of Tempo 2.0 and TraceQL: `https://vimeo.com/773194063`
+
+The video number at the end of the URL.
```
Comment (knylander-grafana) [inline docs/sources/writing-guide/image-guidelines/index.md:282]:
```suggestion

In this example, the video is a Preview of Tempo 2.0 and TraceQL: `https://vimeo.com/773194063`. The video number at the end of the URL.

```
{{< vimeo 773194063 >}}
```

```

**Pull request merged**